Solutions/Okta Single Sign-On/Hunting Queries/ImpersonationSession.yaml

```diff
@@ -1,6 +1,8 @@
 id: 96fb9b37-e2b7-45f6-9b2a-cb9cdfd2b0fc
 name: Initiate impersonation session (Okta) 
 description: |
+    'User.session.impersonation, usually triggered by Okta Support, are rare. This query checks for impersonation events used in LAPSUS$ breach. Review these events and correlate with legitimate Okta support tickets to identify anomalies'
+description-detailed: |
```

Comment:
I feel like you can remove the instruction part form description and keep only in description-detailed, because summarized instructions looks confusing. This is true for all hunting queries. '_Review these events and correlate with legitimate Okta support tickets to identify anomalies_' This is instruction part.

Reply:
Updated , please review it.